edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.Core.Test/DeviceMessageHandlerTest.cs

```diff
@@ -389,6 +390,8 @@ public async Task X509DeviceCanSendMessageTest()
             // send message to x509 device
             Task sendMessageTask = deviceMessageHandler.SendMessageAsync(message, "input1");
             await deviceMessageHandler.ProcessMessageFeedbackAsync(lockToken, FeedbackStatus.Complete);
+
+            await Task.Delay(TimeSpan.FromSeconds(1));
```

Comment:
nit: why not just await sendMessageTask with timeout?

Reply:
If there is a bug in code `sendMessageTask` might never return